Source/WebCore/page/mac/TextIndicatorWindow.mm

```diff
@@ -134,13 +136,41 @@ - (BOOL)isFlipped
     [[m_targetView window] addChildWindow:m_textIndicatorWindow.get() ordered:NSWindowAbove];
     [m_textIndicatorWindow setReleasedWhenClosed:NO];
 
-    if (local_textIndicator->presentationTransition() != TextIndicatorPresentationTransition::None)
+    if (m_textIndicator->presentationTransition() != TextIndicatorPresentationTransition::None)
         [m_textIndicatorLayer present];
 
     if (lifetime == TextIndicatorLifetime::Temporary)
         m_temporaryTextIndicatorTimer.startOneShot(WebCore::timeBeforeFadeStarts);
 }
 
+void TextIndicatorWindow::updateTextIndicator(Ref<TextIndicator> textIndicator, CGRect textBoundingRectInScreenCoordinates)
+{
+    if (m_textIndicator != textIndicator.ptr())
+        m_textIndicator = textIndicator.ptr();
+
+    CGFloat horizontalMargin = dropShadowBlurRadius * 2 + TextIndicator::defaultHorizontalMargin;
+    CGFloat verticalMargin = dropShadowBlurRadius * 2 + TextIndicator::defaultVerticalMargin;
+
+    if (m_textIndicator->wantsBounce()) {
```

Comment:
To fix the smartpointer error:

```suggestion
    bool wantsBounce = textIndicator->wantsBounce();
    if (m_textIndicator != textIndicator.ptr())
        m_textIndicator = WTFMove(textIndicator);

    CGFloat horizontalMargin = dropShadowBlurRadius * 2 + TextIndicator::defaultHorizontalMargin;
    CGFloat verticalMargin = dropShadowBlurRadius * 2 + TextIndicator::defaultVerticalMargin;

    if (wantsBounce) {
```

Reply:
Well, this isn't how I fixed it last time, but whatever.